.github/workflows/testing_minimum.yml

```diff
@@ -8,7 +8,9 @@ name: test minimal version of each dependency
 # see https://github.com/nilearn/nilearn/issues/4069
 
 on:
-
+    pull_request:
+        branches:
+        -   '*'
```

Comment:
probably worth reverting this after merging and making sure that this workflow opens an issue in case of failure so we don't notice it 5 months too late